sequencing/vcf_qc/dxapp.json

```diff
@@ -3,35 +3,108 @@
   "title": "VCF QC",
   "summary": "Apply Recalibration and hard filters to VCF file",
   "dxapi": "1.0.0",
-  "version": "0.0.2",
+  "version": "1.0.0",
   "categories": [
     "Variation Calling"
   ],
   "inputSpec": [
     {
-      "name": "vcf_fn",
+      "name": "fn_vcfgz",
       "label": "VCF File",
       "class": "array:file",
       "optional": false,
-      "patterns": [
-        "*.vcf.gz"
-      ]
+      "patterns": ["*.vcf.gz"],
+      "help": "Array of VCF.gz files."
```

Comment:
"Array of VCF.gz files." -> "Array of VCF files."
Also need to update in the script.